Source/Core/Core/HW/AudioInterface.h

```diff
@@ -22,12 +22,12 @@ bool IsPlaying();
 
 void RegisterMMIO(MMIO::Mapping* mmio, u32 base);
 
-// Get the audio rates (48000 or 32000 only)
-u32 GetAIDSampleRate();
-u32 GetAISSampleRate();
+// Get the audio rate divisors (divisors for 48KHz or 32KHz only)
+u32 GetAIDSampleRateDivisor();
+u32 GetAISSampleRateDivisor();
```

Comment:
Could we put some kind of comment here saying that `Mixer::FIXED_SAMPLE_RATE_DIVIDEND` is the dividend in use? This feels a bit "magic" otherwise.

We could also create a fraction struct and pass around both the dividend and divisor everywhere, but maybe that would be overkill...